src/locales/scripts/create-wp-locale-list.js

```diff
@@ -12,13 +12,17 @@ const { addFetchedTranslationStatus } = require('./get-translations-status')
 
 const base_url =
   'https://raw.githubusercontent.com/GlotPress/GlotPress-WP/develop/locales/locales.php'
+const userAgent =
```

Comment:
I would like to keep the locale scripts independent of each other so that they can be called separately and in any order. To keep this but still not have to configure the UA in each module, I've extracted a `NetworkService` function. The naming is terrible, so I'm open for any suggestions.